manifests/deploy/common-alm-values.yaml

```diff
@@ -1,20 +1,318 @@
 name: metering-operator.vlatest
+labels:
+  operator-metering: "true"
 spec:
   displayName: Metering
   description: Metering can generate reports based on historical usage data from a cluster, providing accountability for how resources have been used.
-  keywords: ['metering', 'metrics', 'reporting', 'coreos']
+  keywords: ['metering', 'metrics', 'reporting']
   version: latest
   maturity: alpha
   maintainers:
-    - name: CoreOS, Inc
-      email: support@coreos.com
+    - name: Red Hat
+      email: sd-operator-metering@redhat.com
   provider:
-    name: CoreOS, Inc
+    name: Red Hat
   labels:
     alm-status-descriptors: metering-operator.vlatest
     alm-owner-metering: metering-operator
   matchLabels:
     alm-owner-metering: metering-operator
+  installModes:
+      - type: OwnNamespace
+        supported: true
+      - type: SingleNamespace
+        supported: false
+      - type: MultiNamespace
+        supported: false
+      - type: AllNamespaces
+        supported: false
+  permissions:
```

Comment:
So permissions is actually not supposed to be set in this file, because we set them somewhere else, and use the output of another file we generate, as the permissions value for this helm chart. It works like this:

So permissions actually is extracted from a different file, a file that is also generated using helm charts. So it works like this:

- https://github.com/operator-framework/operator-metering/blob/master/hack/create-alm-manifests.sh takes the `rules` from the `metering-operator-role.yaml` https://github.com/operator-framework/operator-metering/blob/master/manifests/deploy/openshift/helm-operator/metering-operator-role.yaml#L5 and uses them for the permissions value for this chart when it generates the manifests.
- The `metering-operator-role.yaml` file, is a file generated via the helm-operator chart via this script: https://github.com/operator-framework/operator-metering/blob/master/hack/create-deploy-manifests.sh. create-deploy-manifests.sh uses https://github.com/operator-framework/operator-metering/blob/master/manifests/deploy/common-helm-operator-values.yaml#L30 as the values for the permissions for the helm-operator chart.

So the way that ends up working is:
- make metering-manifests runs `./hack/create-metering-manfests.sh`
- `./hack/create-metering-manfests.sh` runs `./hack/create-deploy-manifests.sh` which uses https://github.com/operator-framework/operator-metering/blob/master/manifests/deploy/common-helm-operator-values.yaml and produces the files in https://github.com/operator-framework/operator-metering/tree/master/manifests/deploy/openshift/helm-operator
- `./hack/create-metering-manfests.sh` then runs `./hack/create-alm-manifests.sh` which uses the https://github.com/operator-framework/operator-metering/blob/master/manifests/deploy/common-alm-values.yaml, and the `manifests/deploy/openshift/helm-operator` files and produces the files in https://github.com/operator-framework/operator-metering/tree/master/manifests/deploy/openshift/alm